.github/workflows/unit-test-on-pull-request.yml

```diff
@@ -16,37 +16,15 @@ jobs:
       matrix:
         go: ["stable"]
     steps:
-    - name: Install dependencies
-      run: sudo apt-get install -y llvm clang dwz cmake curl unzip
-    - name: Install Zydis
-      shell: bash
-      run: |
-        cd /tmp
-        git clone --depth 1 --branch v3.1.0 --recursive https://github.com/zyantific/zydis.git
-        cd zydis
-        rm -rf build
-        mkdir build
-        cd build
-        cmake -DZYDIS_BUILD_EXAMPLES=OFF ..
-        make -j$(nproc)
-        sudo make install
-        cd zycore
-        sudo make install
-    - name: Set up Go ${{matrix.go}}
-      uses: actions/setup-go@v5
-      with:
-        go-version: ${{matrix.go}}
-        check-latest: true
-        cache-dependency-path: |
-          go.sum
-      id: go
-    - name: Check out
-      uses: actions/checkout@v4
-    - name: Linter
-      run: |
-        go version
-        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.56.2
-        make lint
+      - name: Clone code
+        uses: actions/checkout@v4
```

Comment:
We have this repeated at 3 places. I'd consider this as part of setting up the environment.

Reply:
In the setup environment step, a file `.github/workflows/env`, is referenced that is only available if the code is checked out before as it is part of the repository.

Reply:
Ah, that makes sense :)